programs/uxd/src/utils/calculate_depositories_target_redeemable_amount.rs

```diff
@@ -2,38 +2,37 @@ use anchor_lang::prelude::Result;
 use anchor_lang::require;
 
 use crate::error::UxdError;
+use crate::utils::calculate_depositories_sum_value;
 use crate::BPS_UNIT_CONVERSION;
 
 use super::checked_convert_u128_to_u64;
 use super::compute_amount_less_fraction_floor;
 
-pub struct DepositoriesTargets {
-    pub identity_depository_target_amount: u64,
-    pub mercurial_vault_depository_0_target_amount: u64,
-    pub credix_lp_depository_0_target_amount: u64,
+pub struct DepositoriesTargetRedeemableAmount {
+    pub identity_depository_target_redeemable_amount: u64,
+    pub mercurial_vault_depository_0_target_redeemable_amount: u64,
+    pub credix_lp_depository_0_target_redeemable_amount: u64,
```

Comment:
clarification of the names and units. Since we will have multiple complicated maths functions, this helps differenciate everything